docs/essentials/feature-experimentation.md

```diff
@@ -85,4 +85,42 @@ Statistical significance has been achieved if this number is either 0% or 100%.
 ![Positive Metric Results](/feature-experiment-positive-results.png)
 
 **Negative Results**
-![Negative Metric Results](/feature-experiment-negative-results.png)
\ No newline at end of file
+![Negative Metric Results](/feature-experiment-negative-results.png)
+
+
+## Experimentation using a Custom Property for Randomization
+
+:::info
+For documentation on this functionality outside of the context of experimentation you can check out our documentation dedicated to this topic [here](/extras/advanced-targeting/randomize-using-custom-property).
+:::
+
+DevCycle typically uses the User ID as the primary key for Targeting Rules, Feature rollouts and randomization. However, in certain scenarios, Features you release are intended to be rolled out to a cohort of users vs an individual user. For example, a new feature in a B2B platform might impact an entire organization rather than a single user within that organization. In such cases, you can randomize and rollout by using a Custom Property. 
```

Comment:
I think Targeting Rules is  not needed in the first sentence
```suggestion
DevCycle typically uses the User ID as the primary key for Feature rollouts and randomization. However, in certain scenarios, Features you release are intended to be rolled out to a cohort of users vs an individual user. For example, a new feature in a B2B platform might impact an entire organization rather than a single user within that organization. In such cases, you can randomize and rollout by using a Custom Property. 
```